src/schneider_lab_to_nwb/schneider_2024/schneider_2024_metadata.yaml

```diff
@@ -88,6 +88,22 @@ Behavior:
 Sorting:
   units_description: Neural spikes will be sorted offline using Kilosort 2.5 and Phy2 software and manually curated to ensure precise spike time acquisition.
 
+Optogenetics:
+  Device:
+    name: optogenetic_stimulation_laser
+    description: Real time optogenetic stimulation of brain regions of interest will be accomplished via TTL control of an all solidstate 473nm blue laser (MBL-III-473/1~100mW, Opto Engine LLC). Bifurcated fiber cables (ThorLabs, Ø200 µm Core Multimode Fiber) will be used for light delivery.
+    manufacturer: Opto Engine LLC
+  OptogeneticStimulusSite:
+    name: optogenetic_stimulus_site
+    description: To identify cortical neurons that project from the auditory cortex to motor regions (Aim 2), stereotaxic injections of AAV-ChR2 will be made into the primary auditory cortex (-2.8 AP, 4.2 ML relative to bregma; guided by intrinsic optical imaging) during head-fixation and animals will be trained while expression occurs (~2 weeks). In addition, fiber optics will be implanted to target cell bodies in the secondary motor cortex (1.0-1.5 AP, 0.5-0.7 ML).
+    excitation_lambda: 473.0 # nm
+    injection_location: Primary Auditory Cortex  (-2.8 AP, 4.2 ML relative to bregma; guided by intrinsic optical imaging)
+    stimulation_location: Secondary Motor Cortex (1.0-1.5 AP, 0.5-0.7 ML)
+  OptogeneticSeries:
+    name: optogenetic_series
```

Comment:
same here, but I think this illustrates way I personally don't like the convention. When there is not a good name you end up with the same name as the type.